mails/themes/modern/core/order_conf.html.twig

```diff
@@ -50,7 +50,7 @@
                                     <tr>
                                       <td align="left" style="font-size:0px;padding:10px 25px;padding-top:0px;padding-bottom:25px;word-break:break-word;">
                                         <div style="font-family:Open sans, arial, sans-serif;font-size:16px;line-height:25px;text-align:left;color:#363A41;">
-                                          {{ 'Thank you for shopping on [1]{shop_name}[/1]!'|trans({'[1]': '<span class="label">', '[/1]': '</span>'}, 'Emails.Body', locale)|raw }}
+                                          {{ 'Thank you for your order on [1]{shop_name}[/1]!'|trans({'[1]': '<span class="label">', '[/1]': '</span>'}, 'Emails.Body', locale)|raw }}
```

Comment:
Did you make the modification in the mjml sources?

Reply:
Yes, but didn't make a PR for now as I'm not sure this one will be accepted as it?